org.eclipse.transformer.cli/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties

```diff
@@ -25,42 +25,81 @@
 javax.activation=jakarta.activation
 
 javax.annotation=jakarta.annotation
+javax.annotation.concurrent=jakarta.annotation.concurrent
 javax.annotation.security=jakarta.annotation.security
 javax.annotation.sql=jakarta.annotation.sql
+javax.batch=jakarta.batch
+javax.batch.api=jakarta.batch.api
+javax.batch.api.chunk=jakarta.batch.api.chunk
+javax.batch.api.chunk.listener=jakarta.batch.api.chunk.listener
+javax.batch.api.listener=jakarta.batch.api.listener
+javax.batch.api.partition=jakarta.batch.api.partition
+javax.batch.operations=jakarta.batch.operations
+javax.batch.runtime=jakarta.batch.runtime
+javax.batch.runtime.context=jakarta.batch.runtime.context
 
-javax.el=jakarta.el
 com.ibm.wsspi.el=com.ibm.wsspi.el
```

Comment:
This line  should be removed. It is some IBM leftovers...

Reply:
Wasn't sure if you wanted to keep that or not :)

Reply:
Some test cares about this IBM package. So put it back for now. We can remove it and change the test case in another PR.

Reply:
Whoops! I've put that back.